style.css

```diff
@@ -0,0 +1,3 @@
+h1 {

```

Comment:
Even though your implementation works as expected 🔥 Please change your rule selector into a class to meet the project requirements 🤓